ratelimit/README.md

```diff
@@ -2,73 +2,50 @@
 
 Ambassador Pro includes a dynamic rate limiting service that can handle a wide variety of rate limiting use cases. 
 
-## Global Rate Limiting
+## Prerequesites
 
-Suppose we want to limit a certain subset of users to only 10 requests per minute for any requests through Ambassador. We can configure a global rate limit that can rate limit based off a header that identifies this subset of users. Users with the header `x-limited-user: true` will be limited to 10 requests per minute. 
+1. [Install Ambassador Edge Stack](https://www.getambassador.io/early-access/user-guide/getting-started/)
```

Comment:
Should this reference the EA URL or the real URL?

Reply:
Probably real. Thanks!